fabric-data-generation-api-v1/src/main/resources/fabric-data-generation-api-v1.mixins.json

```diff
@@ -3,6 +3,7 @@
   "package": "net.fabricmc.fabric.mixin.datagen",
   "compatibilityLevel": "JAVA_16",
   "mixins": [
+    "loot.BlockLootTableGeneratorMixin",
```

Comment:
Is the package really neeeded for one mixin?

Reply:
Not sure. Since we're gonna expand this module in the future I thought it might be nicer to keep everything organized? The new API file is in a `loot` subpackage (#2754 will also add subpackages), and that should stay like this IMO, but I'm happy to remove the mixin subpackage if you want.

Reply:
Thats fine, if it did remove it its likely it would never be added back. 👍